### PR TITLE
Backend fields

### PR DESCRIPTION
**Pull request by ocmichelleweaver**

### CODE REVIEW

File: wp-content/themes/ocm-base-timber/assets/dist/site.js [hunk 1]
```diff
@@ -1,14 +1,134 @@
 (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
 const $ = require( 'jquery' );
 
-},{"jquery":5}],2:[function(require,module,exports){
+// load functions
```
Comment (mkirshman) [inline wp-content/themes/ocm-base-timber/assets/dist/site.js:4]: The methods(`buttonTabsBlock, basicTextBlock`, etc...) in this file appear to be the same as the methods in block.js Are they needed in both places or can we single source these in one file?

File: wp-content/themes/ocm-base-timber/assets/js/common/blocks.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+const $ = require( 'jquery' );
+
+// load functions
+$(document).ready( function() {
+    buttonTabsBlock();
+    basicTextBlock();
+    basicGridBlock();
+});
+
+function buttonTabsBlock () {
```
Comment (mkirshman) [inline wp-content/themes/ocm-base-timber/assets/js/common/blocks.js:10]: See my comment form `site.js`

File: wp-content/themes/ocm-base-timber/assets/js/common/global.js [hunk 3]
```diff
@@ -2,4 +2,5 @@ const $ = require( 'jquery' );
 
 // load functions
 $(document).ready( function() {
+
```
Comment (mkirshman) [inline wp-content/themes/ocm-base-timber/assets/js/common/global.js:5]: What is `site.js` vs `global.js`?

File: wp-content/themes/ocm-base-timber/assets/scss/abstracts/variables/_structure.scss [hunk 4]
```diff
@@ -32,7 +32,7 @@ $spacing--xxl:                              120px;
  * ---  Variables: Structure - Container Widths
  * -------------------------------------------------------------------- */
 
-$container--width:                          1280px;
+$container--width:                          1024px; //1280px;
```
Comment (mkirshman) [inline wp-content/themes/ocm-base-timber/assets/scss/abstracts/variables/_structure.scss:35]: Go ahead and get rid of the old comment.

File: wp-content/themes/ocm-base-timber/assets/scss/components/blocks/_basic-text.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+        &.truncated {
+            > .inner {
+                max-height: 200px;
+                overflow: hidden;
+            }
+
+            &:after {
+                content: ' ';
+                background: linear-gradient(180deg, rgba(255,255,255,0) 0%, rgba(255,255,255,1) 80%);
```
Comment (mkirshman) [inline wp-content/themes/ocm-base-timber/assets/scss/components/blocks/_basic-text.scss:17]: I know you are setting up colors and such in its own var file, I wonder if this gradient could be added as well.

**Review by mkirshman — CHANGES_REQUESTED** [review]

File: wp-content/themes/ocm-base-timber/assets/dist/site.js (hunk 1 above)
Comment (ocmichelleweaver) [reply wp-content/themes/ocm-base-timber/assets/dist/site.js:4]: This is the compiled JS File that get's loaded into the site.

File: wp-content/themes/ocm-base-timber/assets/js/common/blocks.js (hunk 2 above)
Comment (ocmichelleweaver) [reply wp-content/themes/ocm-base-timber/assets/js/common/blocks.js:10]: All of this compiles into one JS file that get's loaded into the site

File: wp-content/themes/ocm-base-timber/assets/js/common/global.js (hunk 3 above)
Comment (ocmichelleweaver) [reply wp-content/themes/ocm-base-timber/assets/js/common/global.js:5]: site.js get's loaded into the site. Global are all the functions that aren't specific to one component. But I see the confusion if it all get's loaded globally :)

File: wp-content/themes/ocm-base-timber/assets/scss/components/blocks/_basic-text.scss (hunk 5 above)
Comment (ocmichelleweaver) [reply wp-content/themes/ocm-base-timber/assets/scss/components/blocks/_basic-text.scss:17]: Good thought :)

File: wp-content/themes/ocm-base-timber/assets/dist/site.js (hunk 1 above)
Comment (mkirshman) [reply wp-content/themes/ocm-base-timber/assets/dist/site.js:4]: Oh I see.. great! please disregard

File: wp-content/themes/ocm-base-timber/assets/scss/abstracts/variables/_structure.scss (hunk 4 above)
Comment (mkirshman) [reply wp-content/themes/ocm-base-timber/assets/scss/abstracts/variables/_structure.scss:35]: Ok disregard this one I just pinged Jamie and we can address when we get an update

File: wp-content/themes/ocm-base-timber/assets/scss/components/blocks/_basic-text.scss (hunk 5 above)
Comment (mkirshman) [reply wp-content/themes/ocm-base-timber/assets/scss/components/blocks/_basic-text.scss:17]: Cool I think we can approve and merge this one. lets task out future work to make thes pull requests small. I mean, I dont need to review plugins or anything. We can install those and create a PR immediately so we dont hold you up.

**Review by mkirshman — APPROVED** [review]

**Pull request merged**